package.json

```diff
@@ -30,16 +30,13 @@
   "dependencies": {
     "find-up": "^5.0.0",
     "picocolors": "^1.0.0",
-    "postcss": "^8.0.0",
+    "postcss": "^8.3.11",
```

Comment:
Why ^8.3.11 ?

Reply:
Because now this is a dependency not a peerDependency. With the peer dependency we just specify the minimum version needed for the package to work. But here it's a dependency and we already updated all of them.